src/teleop/gui/src/components/ControlGUI.vue

```diff
@@ -2,13 +2,15 @@
 <div>
     <DriveControls></DriveControls>
     <ArmControls></ArmControls>
+    <GimbalControls></GimbalControls>>
 </div>
 
 </template>
 
 <script>
     import DriveControls from './DriveControls.vue';
     import ArmControls from './ArmControls.vue';
+    import GimbalControls from './GimbalControls.vue';
```

Comment:
Please post screenshot of how this looks in browser

Reply:
No front-end changes